.github/workflows/e2e-sample-apps-tests.yml

```diff
@@ -251,8 +257,28 @@ jobs:
             echo "sample_app_branch_exists=false" >> $GITHUB_OUTPUT
           fi
 
-      - name: Prepare and launch Sample Apps
-        if: ${{ steps.check-sample-app-branch.outputs.sample_app_branch_exists == 'true' }}
+      - name: Configure AWS credentials
+        if: ${{ matrix.test_suite == 'shoppy-e2e' }}
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.SHOPPY_IAM_ROLE }}
+          role-session-name: GitHub_to_AWS_via_FederatedOIDC
+          aws-region: us-east-1
+
+      - name: Download Shoppy's App DB PostgreSQL dump from S3
+        if: ${{ matrix.test_suite == 'shoppy-e2e' }}
+        run: |
+          aws s3 cp s3://${{ secrets.SHOPPY_METABASE_APP_DB_S3_BUCKET }}/${{ secrets.SHOPPY_METABASE_APP_DB_S3_KEY }} ./e2e/tmp/db_dumps/shoppy_metabase_app_db_dump.sql
+
+      - name: Prepare and launch Sample Apps for shoppy-e2e test suite
+        if: ${{ steps.check-sample-app-branch.outputs.sample_app_branch_exists == 'true' && matrix.test_suite == 'shoppy-e2e' }}
+        env:
+          METABASE_APP_DB_USER: ${{ secrets.SHOPPY_METABASE_APP_DB_USER }}
+          METABASE_APP_DB: ${{ secrets.SHOPPY_METABASE_APP_DB_DATABASE }}
+        run: npx tsx ./e2e/runner/start-sample-app-containers/start-ci.ts
```

Comment:
nit: I'd rather pass `TEST_SUITE` directly here as a parameter e.g. npx tsx ./e2e/runner/start-sample-app-containers/start-ci.ts shoppy-e2e

This ensures we know that we use the `matrix.test_suite` in the command, otherwise, I need to go to see how `start-ci.ts` is implemented just to see it references `process.env.TEST_SUITE`.

This also removes the implicit dependencies (env.TEST_SUITE) between this workflow file and `start-ci.ts` and rely on direct communication instead which should make the code more clear.

With this we could change the below line to

```bash
npx tsx ./e2e/runner/start-sample-app-containers/start-ci.ts ${{ matrix.test_suite }} # Not quite sure about the string interpolation syntax, but you get the point.
```

Reply:
Done